dfastmi/gui.py

```diff
@@ -604,7 +650,7 @@ def main(rivers_configuration: RiversObject, config: Optional[str] = None) -> No
     rivers = rivers_configuration
     create_dialog()
     dialog["branch"].addItems(rivers["branches"])
-    dialog["reach"].addItems(rivers["reaches"][0])
+    #dialog["reach"].addItems(rivers["reaches"][0])
```

Comment:
Please remove this line if it is not used.

Reply:
Done